### PR TITLE
Integrate Teams meeting hard mute feature part 2 - component bindings

### PR DESCRIPTION
**Pull request by fuyan2024**
# What

# Why

# How Tested

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/types/ParticipantListParticipant.ts [hunk 1]
```diff
@@ -92,3 +95,14 @@ export type ParticipantState =
   | 'EarlyMedia'
   | 'Disconnected'
   | 'Reconnecting';
+
+/* @conditional-compile-remove(media-access) */
+/**
+ * Media access state with order
+ *
+ * @alpha
```
Comment (edwardlee-msft) [inline packages/react-components/src/types/ParticipantListParticipant.ts:103]: You don't need this alpha tag.

**Review by edwardlee-msft — APPROVED** [review]

File: packages/calling-component-bindings/src/handlers/createCommonHandlers.ts [hunk 2]
```diff
@@ -712,6 +729,44 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
...
+    const onPermitParticipantVideo = async (userIds: string[]): Promise<void> => {
+      const participants = userIds?.map((userId) => _toCommunicationIdentifier(userId));
+      await call?.feature(Features.MediaAccess).permitVideo(participants);
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onForbidAllAttendeesVideo = async (): Promise<void> => {
+      await call?.feature(Features.MediaAccess).forbidRemoteParticipantsVideo();
+    };
+    /* @conditional-compile-remove(media-access) */
+    const onPermitAllAttendeesVideo = async (): Promise<void> => {
```
Comment (mgamis-msft) [inline packages/calling-component-bindings/src/handlers/createCommonHandlers.ts:766]: I think we should rename 'onPermitAllAttendeesVideo ' to 'onPermitAllRemoteParticipantsVideo' and likewise for the other handlers like `onMuteAllRemoteParticipants ` to keep it consistent with the other handlers and the Calling SDK.

Comment (fuyan2024) [reply packages/calling-component-bindings/src/handlers/createCommonHandlers.ts:766]: It is not All remote participants, attendees only. 
Agree to keep consistent, will change to onPermitRemoteParticipantsVideo

File: packages/react-components/src/types/ParticipantListParticipant.ts (hunk 1 above)
Comment (fuyan2024) [reply packages/react-components/src/types/ParticipantListParticipant.ts:103]: > You don't need this alpha tag.

It gives compile error without it.

File: packages/calling-component-bindings/src/handlers/createCommonHandlers.ts (hunk 2 above)
Comment (fuyan2024) [reply packages/calling-component-bindings/src/handlers/createCommonHandlers.ts]: Changed it.

**Review by mgamis-msft — APPROVED** [review]

File: packages/calling-component-bindings/src/handlers/createCommonHandlers.ts (hunk 2 above)
Comment (mgamis-msft) [reply packages/calling-component-bindings/src/handlers/createCommonHandlers.ts]: I see. Only participants with attendee role will be affected. I still say to keep it consistent but we just have to document that only attendees will be affected.

**Pull request merged**